src/tooltip/index.html

```diff
@@ -0,0 +1,211 @@
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+	<style>
+		*,
+		*:before,
+		*:after {
+			box-sizing: border-box;
+		}
+
+
+		body {
+			font-family: Arial, sans-serif;
+			padding: 20px;
+			background-color: #f4f4f4;
+		}
+
+		.card-container {
+			display: flex;
+			flex-wrap: wrap; /* Makes cards wrap on smaller screens */
+			justify-content: center; /* Centers the cards horizontally */
+			gap: 20px; /* Adds space between cards */
+		}
+
+		.card {
+			background-color: white;
+			border: 1px solid #ddd;
+			border-radius: 8px;
+			width: 300px; /* Fixed width for cards */
+			box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+			transition: transform 0.3s ease;
+		}
+
+		.card-header {
+			display: flex;
+			padding: 0 20px;
+			justify-content: space-between;
+			align-items: center;
+		}
+
+		.info-icon {
+			color: #555;
+			font-size: 20px;
+			cursor: pointer;
+			transition: color 0.3s ease;
+		}
+
+		.info-icon:hover {
+			color: #007BFF; /* Changes icon color on hover */
+		}
+
+		@media (max-width: 768px) {
+			.card {
+				width: 100%; /* Cards take full width on smaller screens */
+			}
+		}
+	</style>
+</head>
+<body style="padding: 0; margin: 0">
+	<main >
+		<tp-tooltip>
+			<tp-tooltip-trigger>
+				<div class="my-btn">Hover Me</div>
+			</tp-tooltip-trigger>
+			<tp-tooltip-popover style="width: 265px;padding: 5px;border: 2px solid red;">Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia molestiae distinctio tempore nobis rem provident libero voluptatum excepturi optio pariatur, accusantium minus ex, doloremque a, repellendus iure quae eaque consectetur!</tp-tooltip-popover>
+		</tp-tooltip>
+		<p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Doloribus eum nulla quam optio numquam reprehenderit, fugit modi eos pariatur, doloremque magnam consequatur atque officiis necessitatibus, deleniti odit quia commodi ullam. Mollitia, reiciendis ipsum impedit quos nobis molestias sit qui vitae voluptate pariatur, fugiat eveniet, veniam magni tenetur repellat eum. Facere aspernatur nobis laborum fuga dolorem quis fugiat, expedita atque cumque quae quia ut in asperiores? Eum, sunt laudantium beatae voluptas architecto impedit itaque ipsam consequatur, blanditiis atque distinctio corporis consectetur ipsum et ut, adipisci dolore exercitationem harum culpa quia. Id tempora laudantium, amet provident maxime ut est nemo veritatis vel.</p>
+		<tp-tooltip>
+			<tp-tooltip-trigger>
+				<div class="my-btn" style="display: flex;gap: 10px;">
+					<div>Hover me</div>
+					<i>i</i>
+				</div>
+			</tp-tooltip-trigger>
+			<tp-tooltip-popover style="width: 265px;">Lorem ipsum dolor sit amet consectetur adipisicing elit. Mollitia molestiae distinctio tempore nobis rem provident libero voluptatum excepturi optio pariatur, accusantium minus ex, doloremque a, repellendus iure quae eaque consectetur!</tp-tooltip-popover>
```

Comment:
@harshdeep-gill Let's add `<tp-tooltip-arrow></tp-tooltip-arrow>` here so that the user can have full control over the arrow's styles